easybuild/easyblocks/s/scalapack.py

```diff
@@ -56,33 +59,28 @@ def configure_step(self):
         src = os.path.join(self.cfg['start_dir'], 'SLmake.inc.example')
         dest = os.path.join(self.cfg['start_dir'], 'SLmake.inc')
 
-        if not os.path.isfile(src):
-            raise EasyBuildError("Can't fin source file %s", src)
-
         if os.path.exists(dest):
             raise EasyBuildError("Destination file %s exists", dest)
-
-        try:
-            shutil.copy(src, dest)
-        except OSError, err:
-            raise EasyBuildError("Symlinking %s to %s failed: %s", src, dest, err)
+        else:
+            copy_file(src, dest)
 
         self.loosever = LooseVersion(self.version)
 
-        # make sure required dependencies are available
-        deps = [("LAPACK", "ACML", "OpenBLAS")]
-        self.log.deprecated("EB_ScaLAPACK.configure_step uses hardcoded list of LAPACK libs", '3.0')
+        # make sure required dependencies are available for BLAS/LAPACK
+        dep_grps = [('ACML', 'LAPACK', 'OpenBLAS')]
```

Comment:
these are the software names, but I guess I can use `Acml.BLAS_MODULE_NAME[0]` & co here, like I do in #1016?